api/src/test/java/org/openmrs/test/jupiter/BaseContextSensitiveTest.java

```diff
@@ -119,7 +119,7 @@
 @ExtendWith(MockitoExtension.class)
 public abstract class BaseContextSensitiveTest {
 	
-	private static final Logger log = LoggerFactory.getLogger(BaseContextSensitiveTest.class);
+	public static final Logger log = LoggerFactory.getLogger(BaseContextSensitiveTest.class);
```

Comment:
Please don't change this visibility if you need a logger in your test, create a class-specific version.